ado_asana_sync/sync/__main__.py

```diff
@@ -1,10 +1,10 @@
 import logging
 from ado_asana_sync.sync.sync import read_projects, sync_project
-from ado_asana_sync.sync.app import app
+from ado_asana_sync.sync.app import App
 
 # Create an instance of the app configuration and connect to the services.
 logging.basicConfig(level=logging.INFO)
-app_config = app()
+app_config = App()
 app_config.connect()
```

Comment:
**Suggestion:** Consider handling exceptions that may occur when connecting to the services. This will improve the robustness of your code.
```suggestion
app_config = App()
try:
    app_config.connect()
except Exception as e:
    logging.error(f'Failed to connect: {e}')
```